### ISSUE TITLE
Version 1.2.0 and SpecFlow Version 2.0.0

### ISSUE THREAD

**Issue by CarlConn** [baseclass/Contrib.SpecFlow.Selenium.NUnit#14]
BaseClass.Contrib.SpecFlow.Selenium.Nunit version 1.3.1 has a dependencies of SpecFlow version 1.9.0. After installing that version, I get Build errors. I install SpecFlow version 2.0.0 and now getting this error...
"#error Generation error: Could not load file or assembly 'TechTalk.SpecFlow, Version=1.9.0.77, Culture=neutral, PublicKeyToken=0778194805d6db41' or one of its dependencies. The system cannot find the file specified."
It appears to me that the BaseClass.Contrib needs to be updated or allow updates to its dependencies. Am I correct?
Bottom line, I am having issues with the BaseClass.Contrib that I am getting from NuGet.
Thank you.
Carl

**Comment by unickq** [comment]
That's because BaseClass.Contrib.SpecFlow.Selenium.Nunit was built for Specflow 1.9, not 2.0
